opencti-platform/opencti-front/src/components/dataGrid/components/DataTableBody.tsx

```diff
@@ -26,21 +27,27 @@ const DataTableBody = ({
     endsWithAction,
     actions,
     columns,
-    useDataTable: {
-      data: queryData,
-      isLoading,
-      loadMore,
-      hasMore,
-    },
+    dataQueryArgs,
     useDataTableToggle: {
       selectedElements,
       onToggleEntity,
     },
     useDataTablePaginationLocalStorage: {
       viewStorage: { filters },
     },
+    data,
   } = useDataTableContext();
 
+  const {
+    data: paginatedData,
+    isLoading,
+    loadMore,
+    hasMore,
+  } = useDataTable(dataQueryArgs);
+
+  // data is from datatableWithoutFragment
+  const queryData = data ?? paginatedData;
+
```

Comment:
In case we only use data (DataTableWithoutFragment) calling useDataTable doesn't work. 
```suggestion
  const {
    data: queryData,
    isLoading,
    loadMore,
    hasMore,
  } = data ? { data } : useDataTable(dataQueryArgs);
```